lib/index.js

```diff
@@ -167,9 +168,11 @@ internals.addMethod('length', function (size) {
 
 internals.addMethod(['equal', 'equals'], function (value, options) {
 
+    options = options || {};
+    var settings = Hoek.applyToDefaults(exports.settings, options);
```

Comment:
We don't want to pass all of `exports.settings` to Hoek. The defaults can be `{ prototype: exports.settings.prototype }`.